components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -194,4 +194,11 @@ export class ProjectListComponent implements OnInit {
   cancelApplyStop() {
     this.closeConfirmApplyStopModal();
   }
+
+  getRulesStatus(project: Project): string {
+    return project.status === 'NO_RULES' ? 'No rules'
```

Comment:
Let's use a switch here, please 😉

Reply:
My preference is for functional-style code, which requires a ternary as given rather than a switch. However, sounds like your preference is strong, so I am happy to oblige.